AncientReligions/interface/coats_of_arms.txt

```diff
@@ -116,7 +116,7 @@ culture =
 		texture = {
 			file = "gfx\\coats_of_arms\\dynasties4.tga"
 			size = { x = 16 y = 4 }
-			noOfFrames = 57
+			noOfFrames = 52
 			color = 0
```

Comment:
I've found the cause of shifted religious icons that I reproduce with your master branch. I'll do a separate pull request.